hasura.planx.uk/metadata/tables.yaml

```diff
@@ -485,6 +485,15 @@
           table:
             name: published_flows
             schema: public
+    - name: template_children
```

Comment:
Not convinced that "children" is really the right term here - very open to suggestions.

Reply:
Are they `templated_flows` (again following `published_flows` pattern)?

Many published flows or templated flows to one "source" flow is my evolving mental model?

Reply:
Much better!